components/TextFields/src/MDCBaseTextField.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Copyright headers are out of date; please update.

Reply:
I always do this >_<

Reply:
Done in e476a2c